pack-plugin.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+if (! class_exists(ZipArchive::class)) {
+    echo 'This script requires zip PHP extension.' . PHP_EOL;
+    exit(1);
+}
+
+$plugin = $argv[1] ?? null;
+
+$directory = __DIR__ . '/plugins/' . $plugin . '/src';
+
+if (! $plugin || ! preg_match('~^(?:[a-z-_]++)$~', $plugin) || ! is_dir($directory)) {
+    echo 'Plugin name was not specified or is invalid.' . PHP_EOL;
+    exit(2);
+}
+
+chdir($directory);
+
+if (file_exists($directory . '/composer.json')) {
```

Comment:
extend the test for PRs to check whether there is `compose.lock` file if `composer.json` is present

Reply:
Perhaps to https://github.com/Ubiquiti-App/UCRM-plugins/pull/81

Reply:
https://github.com/Ubiquiti-App/UCRM-plugins/pull/81/commits/946522f3e688b4d486b4006691223b5eba97cb22